ui/packages/consul-ui/app/components/consul/auth-method/type/index.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="type {{@item.Type}}" data-test-type={{@item.Type}}>
```

Comment:
I think the class here should be `consul-auth-method-type` to stick with the "component name = classname" pattern. I think eventually we'll have a `Badge` component and these (and the other 2 similar components) will go.